ab2d-events-client/build.gradle

```diff
@@ -10,9 +10,12 @@ dependencies {
     implementation "com.fasterxml.jackson.datatype:jackson-datatype-jdk8:${jacksonVersion}"
     implementation "com.fasterxml.jackson.datatype:jackson-datatype-jsr310:${jacksonVersion}"
     implementation "com.slack.api:slack-api-client:${slackAPIVersion}"
-    implementation "org.projectlombok:lombok:${lombokVersion}"
```

Comment:
Lambok was being added to the fat jar and passed down the chain. Normally no one would care but it's inflating the size of the lambda zips.